paasta_tools/kubernetes_tools.py

```diff
@@ -2154,7 +2198,10 @@ def has_routable_ip(
         return "false"
 
     def get_pod_template_spec(
-        self, git_sha: str, system_paasta_config: SystemPaastaConfig
+        self,
+        git_sha: str,
+        system_paasta_config: SystemPaastaConfig,
+        restart: bool = True,
```

Comment:
non-blocking suggestion: `restart_on_failure` might be a slightly more descriptive name